project-koku/koku
# Ocp gcp credits

Fixes [COST-1026]

Changes proposed in this PR:
* The problem is when processing nise files we pass in the parquet_base_filename as the last invoice month file that was processed in convert_csv_to_parquet. This then causes create_ocp_on_cloud_parquet to only correlate data for the current month.
* The Change here updates the file name convention for each invoice month based on the data_frame 

Testing Instructions:
1. Add OCP GCP data (Using bigquery)
2. Check we have data at 
   GET http://localhost:8000/api/cost-management/v1/reports/openshift/infrastructures/gcp/costs/

---
Additional Context:
